### PR TITLE
Added INFS4205 review

### PR DESCRIPTION
**Pull request by jack775544**
Added INFS4205 course review. Haven't checked if it can compile since LaTeX hasn't finished downloading yet. (I mean 3GB, really?)

### CODE REVIEW

File: tex/courses/subjects/INFS4205.tex [hunk 1]
```diff
@@ -2,16 +2,26 @@
...
 title = {Advanced Techniques for High Dimensional Data},
-score = 4,
+score = 5,
 prereq = {INFS2200},
 contact = {2L},
 coordinator = {Prof Xiaofang Zhou (zxf@itee.uq.edu.au)},
 assessment = {
-example & example\% & example \\
+Assignment 1 & 15\% & Theory Assignment \\
+Assignment 1 & 15\% & Group Assignment \\
```
Comment (wisebaldone) [inline tex/courses/subjects/INFS4205.tex]: Is their 2 parts to assignment 1?

Comment (wisebaldone) [reply tex/courses/subjects/INFS4205.tex]: This also doesn't add up to 100%

File: tex/courses/subjects/INFS4205.tex [hunk 2]
```diff
@@ -2,16 +2,26 @@
...
+
+    The only contact hours for the subject are weekly 2 hour lectures. While attendance is not required, class discussion usually goes on in them, meaning that turning up can be quite beneficial.\\
+
+    There are 2 assignments with the first being a relatively easy theory assignment and the second being a group assignment. The first assignment is just several theory questions that you must answer, where the second assignment involves choosing a project idea relating to the course content and working on that. Examples of this are data driven software projects powered by a spatial database, to in-depth theoretical analysis of algorithms that were taught in the course. The final submission for this assignment is a presentation and research report.\\
+
+    The final exam is open book and can be quite difficult. Even though the theory behind it is fairly simple as long as you are familiar with the course content, the questions require in depth answers that can take a large amount of time to answer. This means that the main challenge in the exam is making small mistakes on rushed answers, rather then trouble with the overall theory.
```
Comment (wisebaldone) [inline tex/courses/subjects/INFS4205.tex]: `\\` at the end of this line pls boss

File: tex/courses/subjects/INFS4205.tex (hunk 1 above)
Comment (jack775544) [reply tex/courses/subjects/INFS4205.tex]: Copy pasta too hard. Fixed now

File: tex/courses/subjects/INFS4205.tex (hunk 2 above)
Comment (jack775544) [reply tex/courses/subjects/INFS4205.tex]: We wouldn't have this issue with markdown :/
Fixed

File: tex/courses/subjects/INFS4205.tex [hunk 3]
```diff
@@ -2,16 +2,26 @@
...
 coordinator = {Prof Xiaofang Zhou (zxf@itee.uq.edu.au)},
 assessment = {
-example & example\% & example \\
+Assignment 1 & 25\% & Theory Assignment \\
+Assignment 2 & 25\% & Group Assignment \\
+Final Exam & 55\% & Open book short answer exam \\
 },
 review = {
-    review here
+    This course provides an overview on the of the theory of high dimensional databases, with a focus on indexing and searching. This content builds on the theory taught in INFS2200 and it is highly recommended to have taken that course in the past. In short, this course continues to build on the indexing and query planning parts of INFS2200.\\
```
Comment (draganmarjanovic) [inline tex/courses/subjects/INFS4205.tex]: "on the of the" rephrase

Comment (draganmarjanovic) [reply tex/courses/subjects/INFS4205.tex]: "components of INFS2200" sounds better IMO

**Review by draganmarjanovic — CHANGES_REQUESTED** [review]
Parts of line 15 need rewording IMO.

**Comment by wisebaldone** [comment]
@draganmarjanovic can you re-review the changes

File: tex/courses/subjects/INFS4205.tex [hunk 4]
```diff
@@ -9,19 +9,19 @@
...
 
-    The final exam is open book and can be quite difficult. Even though the theory behind it is fairly simple as long as you are familiar with the course content, the questions require in depth answers that can take a large amount of time to answer. This means that the main challenge in the exam is making small mistakes on rushed answers, rather then trouble with the overall theory.\\
+    The final exam is open book and can be quite difficult. Even though the theory behind it is fairly simple, as long as you are familiar with the course content, the questions require in depth answers that can take a large amount of time to answer. This means that the main challenge in the exam is making small mistakes on rushed answers, rather then trouble with the overall theory.\\
```
Comment (draganmarjanovic) [inline tex/courses/subjects/INFS4205.tex]: " as long as you are familiar with the course content" should either be in brackets IMO or completely removed.

**Review by draganmarjanovic — CHANGES_REQUESTED** [review]
Aside from this - all good.

**Comment by jack775544** [comment]
I think it is all good now

**Comment by wisebaldone** [comment]
LGTM, im taking Dragan as one piece of three

**Comment by draganmarjanovic** [comment]
👍

**Pull request merged**